daml-model/daml/Model/LoanConfig.daml

```diff
@@ -0,0 +1,65 @@
+--
+-- Copyright (c) 2022, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+--
+
+module Model.LoanConfig where
+
+import Model.Balance
+import DA.Optional
+
+type Amount = Decimal
+type IBAN = Text
+
+-- Loan configuration on account (e.g. IBAN) level
+-- The idea is to provide the {accountHolder} with individual loans up {maxLoanPerTransaction},
+-- as long as {maxLoanThreshold} has not been reached (as tracked by {activeLoan}).
+template LoanConfig
+    with
+        provider: Party
+        -- ^ Account provider
+        accountHolder: Party
+        -- ^ Account holder
+        account: IBAN
+        -- ^ Account number
+        maxLoanPerTransaction: Amount
+        -- ^ Defines the limit of a single loan
+        maxLoanThreshold: Amount
+        -- ^ Defines the limit of all loans given to this account
+        activeLoan: Amount
+        -- ^ Sum of currently active loans
+        paybackThreshold: Optional Amount
+        -- ^ Defines at which balance amount the loan will be paid back
+
+    where
+        -- TODO: multisig (propose/accept)
+        -- signatory provider, accountHolder
```

Comment:
I think we may want to ask Darko how exactly loans are configured aka. how a LoanConfig is created (propose/accept and multisig provider+accountHolder?). @darko-da

Reply:
He suggested to skip for now.